packages/api/cms-api/src/generator/generate-crud-input-json.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { BaseEntity, Entity, MikroORM, PrimaryKey, Property } from "@mikro-orm/core";
+import { LazyMetadataStorage } from "@nestjs/graphql/dist/schema-builder/storages/lazy-metadata.storage";
+import { v4 as uuid } from "uuid";
+
+import { generateCrudInput } from "./generate-crud-input";
+import { lintSource, parseSource } from "./utils/test-helper";
+
+@Entity()
+export class TestEntityWithJson extends BaseEntity<TestEntityWithJson, "id"> {
+    @PrimaryKey({ type: "uuid" })
+    id: string = uuid();
+
+    @Property({ type: "json" })
+    foo: string[] = [];
+
+    @Property({ type: "json" })
+    bar?: number[] = [];
+}
+
+describe("GenerateCrudInputJson", () => {
+    describe("string input class", () => {
```

Comment:
These tests only test parts of the new feature. For instance, embedded properties or custom classes are not tested. Why did you decide to only test literals?

However, some tests are better than no tests at all, thanks 👍🏼

Reply:
I added more tests